frontend/src/screens/Todo.tsx

```diff
@@ -90,8 +90,7 @@ function Todo() {
         </Table>
       </Form>
       <Button
-        size="sm"
-        variant="primary"
+        // Checkpoint 1: Modify Button's `size` and `variant` props
```

Comment:
Just wondering, do we want to include a comment where they can find what kind of props are passable? E.g.: 
// Refer to docs: https://react.designsystem.tech.gov.sg/iframe.html?viewMode=docs&id=components-button--button-with-icon 
Not sure if that will be more helpful or too much handholding haha

Reply:
sgtm! there could be different places to look for documentation, good to set context on how to self help